### PR TITLE
Add YT popularity score and handle DB migrations

### PR DESCRIPTION
**Pull request by cipherLord**
Modified `c3po/db/dao/song.py` and `c3po/job/utils/metadata.py` to incorporate the YT views and release date received from SongData. 
Further, calculated the popularity score and added it to the song table.

> Here's a preview of the current song table

  
Screenshot from 2020-05-27 17-17-08

### CODE REVIEW

File: c3po/db/dao/song.py [hunk 1]
```diff
@@ -36,6 +36,7 @@ class Song(Base):
     explicit = Column("explicit", Boolean)
     popularity = Column("popularity", Float)
     image = Column("Image", String(160))
+    yt_popularity = Column("yt_popularity", Float)
```
Comment (ghostwriternr) [inline c3po/db/dao/song.py:39]: Can this field be renamed to something more generic like `custom_popularity`?

File: c3po/job/utils/metadata.py [hunk 2]
```diff
@@ -60,19 +60,25 @@ def _insert_link(url, session):
...
         date = datetime.strptime(track_data.year, "%Y-%m-%d")
     except ValueError:
         date = datetime.strptime(track_data.year, "%Y")
     except BaseException:
         date = None
+
+    delta = datetime.now() - extras['youtube']['posted_date']
+    factor = 24*60*60
+    score = float(extras['youtube']['views'] /(delta.days * factor))
+
```
Comment (ghostwriternr) [inline c3po/job/utils/metadata.py:74]: Extract this into a separate method like `get_custom_popularity` please.

File: c3po/job/utils/metadata.py [hunk 3]
```diff
@@ -60,19 +60,25 @@ def _insert_link(url, session):
...
     new_song = Song(
         track_data.name,
         date,
         track_data.explicit,
         track_data.popularity,
         track_data.image_id,
+        score,
```
Comment (ghostwriternr) [inline c3po/job/utils/metadata.py:81]: Use the same variable name as the column name so it is always very clear what is happening.

**Comment by cipherLord** [comment]
@ghostwriternr I have fixed the issues. Check once.

**Pull request merged**